public/css/main.css

```diff
@@ -35,3 +35,27 @@ h1 {
 .pr-item {
   word-break: break-word;
 }
+
+.u-bgc--f4f4f4 {
+  background-color: #f4f4f4;
+}
+.u-overflow--hidden {
+  overflow: hidden;
+}
+.u-opacity--disable {
+  opacity: 0;
+}
+.u-opacity--enable {
+  opacity: 1;
+}
+
+.Image {
+  width: auto;
+}
+.Image--small {
+  width: 128px;
+  height: 128px;
+}
```

Comment:
Can you use some of the existing width/height classes https://github.com/tachyons-css/tachyons-widths https://github.com/tachyons-css/tachyons-heights ?